krun.py

```diff
@@ -265,7 +265,11 @@ def inner_main(mailer, on_first_invocation, config, args):
             util.fatal(error_msg)
 
         debug("Using pre-recorded initial temperature readings")
-        platform.starting_temperatures = current.starting_temperatures
+        manifest = ManifestManager(config, platform)
+        platform.starting_temperatures = {
+            sensor: tup[1] for sensor, tup in
+            manifest.starting_temperatures.iteritems()
```

Comment:
I agree with Sarah: this is really confusing to read at the moment. [I must admit that, seeing it for the second time, I think that I dislike the dict comprehension syntax, though that's a secondary issue.]